Racing/PlayerCar.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Racing
+{
+    public class PlayerCar
+    {
+        public int СarLeftPositionX { get; private set; }
+
+        public int СarRightPositionX { get; private set; }
+
+        public int СarPositionY1 { get; private set; }
+
+        public int СarPositionY2 { get; private set; }
+
+        public int CurrentPositionX { get; private set; }
+
+        public int CarHeight { get; private set; }
+
+        public PlayerCar()
+        {
+            this.CurrentPositionX = -1;
+            this.СarLeftPositionX = Config.PlayerCarLeftPositionX;
+            this.СarRightPositionX = Config.PlayerCarRightPositionX;
+            this.СarPositionY1 = Config.PlayerCarPositionY1;
+            this.СarPositionY2 = Config.PlayerCarPositionY2;
+            this.CarHeight = Config.CarHeight;
+        }
+
+        public void MoveLeft()
+        {
+            if (this.CurrentPositionX != this.СarLeftPositionX)
+            {
+                this.CurrentPositionX = this.СarLeftPositionX;
+
+                this.ClearOldCarPosition(this.СarRightPositionX, this.СarPositionY1);
```

Comment:
The Print class only should know about Console.
Car class is responsible for movement, not drawing.

Reply:
-2 to single responsibility principle.